openbas-front/src/admin/components/common/form/TextFieldAskAI.tsx

```diff
@@ -418,7 +412,7 @@ const TextFieldAskAI: FunctionComponent<TextFieldAskAiProps> = ({
               {t('Cancel')}
             </Button>
             <Button
-              disabled={messageInput.length < 5}
+              disabled={currentValue.length === 0}
```

Comment:
Why here it's currentValue.lenght et not messageInput.length ?

Reply:
Because this tone generator corresponds to the main menu, the action should be applied to the currentMessage and not to the MessageInput (which is the text field used when you want to generate text).

Reply:
Oh great !
Thanks for the explanation